### PR TITLE
Added descriptions for home page

### LINKED ISSUE #39: Content: 4 short descriptions needed to add in homepage.
## Description
As a Developer,
I need to have 4 descriptions (getting started, ways to contribute, our projects, open source programs),
so that I can place them in homepage following the agreed mockup.

## Mocks
[Agreed mockup](https://anitab-org.zulipchat.com/user_uploads/16723/uvNAUxnyTGnUq1hhHCNiNJFU/AnitaB_OS_Website-1st-Iteration.pdf)

### Update [Required]
- [ ] `getting started` short descriptions
- [ ] `ways to contribute` short descriptions
- [ ] `our projects` short descriptions
- [ ] `open source programs` short descriptions

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[3] hours

### PR DESCRIPTION
**Pull request by vj-codes**
### Description
Added content for home page 
Fixes #39 

- Documentation

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by vj-codes** [comment]
@annabauza can u have a look at this now?I have made the requested changes.Sorry closed the last PR as messed up something wont do it next time.

File: src/home_content.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
+{ 
+	"sections" : [
+		{ 
+			"title": "Getting Started" ,
+			"content" :  [
+			    { 
+					"par": "The AnitaB.org community is a fusion of rich values and diverse cultures consisting of members all across the seven oceans with a fearless mission of inspiring ,motivating and helping women with their involvement in technical fields and has made a significant impact with regards to contribution of women in tech.\n The other values of the organization include Accountable,Agile,Curious,Courageous,&Diversity,Equity,Inclusion (DEI),encourage everyone to follow the org’s contributing guidelines and obviously passion for open source. "
```
Comment (annabauza) [inline src/home_content.json:7]: please change `&Diversity` to `Diversity`

File: src/home_content.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+				},
+				{
+					"par": "Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.\n The #announcements stream will be displaying all the important announcements.Tag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."
```
Comment (annabauza) [inline src/home_content.json:11]: please change `stream.Zulip` to `stream. Zulip`

File: src/home_content.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+				},
+				{  
+					"par": "Our code of conduct is simple- 'Be nice,Be respectful' which is the one of the many reasons the community coexists in harmony .The members should contribute actively keeping the policies of being repectful to everyone and refrainging from conflicts, unruly behaviour and speech.Anyone who wishes to join the community is expected to follow these guidelines.\n If you are subject to or witness unacceptable behavior, or have any other concerns,please notify a community organizer or adminas soon as possible."
```
Comment (annabauza) [inline src/home_content.json:14]: please change `Be nice,Be respectful` to `Be nice, be respectful`

File: src/home_content.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+			
+				}
+			]
+		},
+	    {
+			"title": "Ways To Contribute",
+			"content":[
+				{
+					"par": "You can start contributing to our various open source projects by-1)Creating issues on Github regarding the project 2)Debugging the issues 3)Creating pull requests, testing pull requests 4)Create mockups for new applications or redesign the current features 5)Research about UI/UX improvements and accessibilty features. 6)Create app walkthroughs with documents or videos,UI testing once the PR is submitted for the UI issue"
```
Comment (annabauza) [inline src/home_content.json:23]: please add \n before every number 1) and add space after number brackets, so ` 4)Create` should be `\n4) Create`.

File: src/home_content.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+				},
+				{
+					"par": "Each active repository has a stream to direct questions.Issues labeled as 'First Timers Only' are meant for contributors who have not contributed to the project yet.Please choose other issues to contribute to, if you have already contributed to these type of issues.Make sure to follow the Commit Message Style Guide when submitting PRs which will require review by atleast one maintainer to be merged to the main code."
```
Comment (annabauza) [inline src/home_content.json:27]: please replace `issues.Make` with `issues. Make`

File: src/home_content.json [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+				}
+			]
+		},
+		{
+			"title": "Our Projects",
+			"content":[
+				{
+					"par": "First and foremost,choose a programming language of your choice.Once you are done with selecting a programming language search for a project .Here's a list of our open source projects - "
```
Comment (annabauza) [inline src/home_content.json:35]: please replace ` project .Here's` with ` project. Here's`

File: src/home_content.json [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+				},
+				{
+					"par": "1)Mentorship System \n Mentorship System is an application that matches women in tech to mentor each other, on career development, through  relations during a certain period of time. \n It is an open source project proposed for Google Summer of Code 2018 with AnitaB.org Open Source,an idea that consists of a mobile application that matches women to help and support both parties. One is the mentor that will be a support/accountability buddy and or give advice to a woman that wants mentorship, for a period of time established by both parties.This would not focus only on students, this would target women from any age, background, work in tech experience, and so on.This project uses Kotlin for frontend and Python for backend. \n Flutter and iOS versions are also available."
```
Comment (annabauza) [inline src/home_content.json:38]: please replace `1)Mentorship` with `1) Mentorship`, there should be no space after `\n`

File: src/home_content.json [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+				},
+				{
+					"par": "2)The Volunteer Management System (VMS) \n The Volunteer Management System (VMS) will be useful for volunteer coordinators, volunteer sign-up,tracking hours, and reporting .This project is currently under development same as the mentorship application project.The Volunteer Management System will be developed in phases. The initial implementation will track  volunteers’ contact information, enable administrators to track volunteer hours, and allow reporting of useful information.This project uses the Django web framework and Python. "
```
Comment (annabauza) [inline src/home_content.json:42]: same as above

File: src/home_content.json [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+						   
+				},
+				{
+					"par": "3)PowerUp Android \n PowerUp is a female empowerment educational mobile game app that will allow young girls to learn about reproductive health and self-esteem by navigating the life of their avatar.This project uses Java.Android and iOS versions are available."
```
Comment (annabauza) [inline src/home_content.json:46]: same as above

File: src/home_content.json [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+				},
+				{
+					"par": "4)AnitaB.org.github.io \n AnitaB.org Open Source web gathering all informations about current open source projects, running programs and events as well as all necessary contribution informations."
+					
+				},
+				{
+					"par": "5)Bridge-in-tech \n  Bridge-In-Tech is an application inspired by the existing AnitaB.org Mentorship System. It encourages organizations to collaborate with the mentors and mentees on mentoring programs. Through Bridge-In-Tech,an organization can offer a mentorship program to a mentor and a mentee that is customised to meet the needed skills set within its organisation while providing a safety and supportive environment for these mentor/mentee to work in.This project was proposed as an original project of Google Summer of Code 2020.The frontend of this application will be a Web application using (tech stack tba)."
```
Comment (annabauza) [inline src/home_content.json:53]: same as above

File: src/home_content.json [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+				{
+					"par": "And more interesting and informative projects in future!"
+				}
+			]
+		},
+		{
+			"title": "Open Source Programs ",
+			"content" : [
+				{
+					"par": "1)Google Summer of Code \n AnitaB.org has been actively participating in the Google Summer of Code in the years 2016,2017,2018 and is one of the most successful organizations since the very beginning.Here's what participants shared about their experience with AnitaB.org -\n Mohammad Murad , former AnitaB.org GSoC mentor in his Medium blog says- AnitaB.org is very different , along with writing coding equal importance is given to interacting with the community .The community also helps in personal development and personal growth.\n Isabel Costa , GSoC 2018 participant in her Medium blog says- ' I got to meet incredible people from the AnitaB.org community .I grew personally and professionally by sharing and learning from other’s perspectives.Being recognized for my work as the Most Outstanding Student by Systers made me very proud of my achievements.This was truly an awesome experience."
```
Comment (annabauza) [inline src/home_content.json:65]: same as above

File: src/home_content.json [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+				}, 
+				{
+					"par": "2)Google Code In \n AnitaB.org has been actively participating in the Google Summer of Code in the years 2014,2015 ,2016,2017,2019 and is one of the most successful organizations since the very beginning.Here's what participants shared about their experience with AnitaB.org - \n Vatsal Kulshrestha , former GCI 2017 participant in his Medium blog says-“AnitaB.org has been a true personification of the famous quote by Einstein- ‘the only source of nowledge is EXPERIENCE’ "
```
Comment (annabauza) [inline src/home_content.json:68]: same as above + please replace `2014,2015 ,2016,2017,2019` to `2014, 2015, 2016, 2017, 2019`.

File: src/home_content.json [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+						
+				}
+			]		
+				
+		}	
+	]
+}			
+			
+
```
Comment (annabauza) [inline src/home_content.json:77]: we need only 1 new line afrer text finish

File: src/home_content.json (hunk 3 above)
Comment (isabelcosta) [inline src/home_content.json:14]:
```suggestion
					"par": "Our code of conduct is simple- 'Be nice,Be respectful' which is the one of the many reasons the community coexists in harmony .The members should contribute actively keeping the policies of being respectful to everyone and refraining from conflicts, unruly behaviour and speech. Anyone who wishes to join the community is expected to follow these guidelines.\n If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or admins as soon as possible."
```

File: src/home_content.json [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+				},
+				{
+					"par": "2)The Volunteer Management System (VMS) \n The Volunteer Management System (VMS) will be useful for volunteer coordinators, volunteer sign-up,tracking hours, and reporting .This project is currently under development same as the mentorship application project.The Volunteer Management System will be developed in phases. The initial implementation will track  volunteers’ contact information, enable administrators to track volunteer hours, and allow reporting of useful information.This project uses the Django web framework and Python. "
+						   
+				},
+				{
+					"par": "3)PowerUp Android \n PowerUp is a female empowerment educational mobile game app that will allow young girls to learn about reproductive health and self-esteem by navigating the life of their avatar.This project uses Java.Android and iOS versions are available."
+				},
+				{
+					"par": "4)AnitaB.org.github.io \n AnitaB.org Open Source web gathering all informations about current open source projects, running programs and events as well as all necessary contribution informations."
```
Comment (isabelcosta) [inline src/home_content.json:49]:
```suggestion
					"par": "4)AnitaB-org.github.io \n AnitaB.org Open Source web gathering all informations about current open source projects, running programs and events as well as all necessary contribution informations."
```

File: src/home_content.json (hunk 3 above)
Comment (vj-codes) [reply src/home_content.json:14]: Thanks @isabelcosta .I'll make the changes soon.Just a question , I didnt get what changes have you requested here.Could you please explain .Thanks in advance:)

**Comment by vj-codes** [comment]
> This is great, just some minor comments on punctuations.

thanks @annabauza .I'll do the changes .Can you  help me about how to update the existing pull

> This is great, just some minor comments on punctuations.

Thanks @annabauza .I'll do the changes and update soon.
I haven't updated a PR before so when I will  push the changes will it be updated automatically or do I need to do something more?

**Comment by vj-codes** [comment]
> This is great, just some minor comments on punctuations.

Thanks @annabauza .I'll do the changes and update soon.
I haven't updated a PR before so when I push the changes will it get updated automatically or do I need to do something more?

**Comment by annabauza** [comment]
@vj-codes how it's going?

**Comment by vj-codes** [comment]
> @vj-codes how it's going?

The other changes are done , I was waiting for Isabel's reply for a change which I haven't understood.Can u help me?

File: src/home_content.json (hunk 3 above)
Comment (vj-codes) [reply src/home_content.json:14]: > this one@annabauza

Comment (vj-codes) [reply src/home_content.json:14]: this one @annabauza

**Comment by annabauza** [comment]
@vj-codes I don't see you to align my comments...

**Comment by vj-codes** [comment]
> @vj-codes I don't see you to align my comments...

sure!

**Comment by vj-codes** [comment]
@annabauza I have done the requested changes , please have a look now:)

**Pull request merged**